ee/tabby-webserver/src/schema/github_repository_provider.rs

```diff
@@ -11,6 +11,8 @@ pub struct GithubRepositoryProvider {
     pub id: ID,
     pub display_name: String,
     pub application_id: String,
+    #[graphql(skip)]
+    pub access_token: Option<String>,
```

Comment:
```suggestion
    pub access_token: String,
```

Reply:
The access token can be `None`, and this will only be used internally. I think we should prefer a closer representation of the data, since otherwise I'll be having to check `if access_token.is_empty()` instead.